### ISSUE TITLE
[Bug]: (visual) queue may get stuck with one object in it

### ISSUE THREAD

**Issue by aus2004** [FuryFight3r/ScrappingSimulator#70]
### Describe the bug

sometimes when you're queuing things a single thing will stay in the queue.
There's also a thing going on that's unrelated where it'll take the wrong item from the queue.

i lost my image so have fun figuring that out i guess

**Comment by FuryFight3r** [comment]
Hi there Aus,
Thank you for reporting this, I think it may be time also to overhaul the Queuing System, as players and I had noticed this upon introducing, but all attempts to fix were unsuccessful, so as a temp fix I made it so the Queue gets completely reset and hidden when exiting Disassembly mode.
A simple band-aid solution to this may be to just run a simple loop check maybe every 1 second that checks: if is not disassembling and queue > or = 1 then reset the queue to 0.
Though in the end I would like to rework the Queuing System either way. 🙂
